extensions/amp-video/0.1/amp-video.js

```diff
@@ -768,6 +800,18 @@ export class AmpVideo extends AMP.BaseElement {
     return this.element.classList.contains('i-amphtml-poolbound');
   }
 
+  /**
+   * Whether the video is just a placeholder and meant to be overriden
+   * @param {!Element} videoEl
+   * @return {boolean}
+   */
+  isPlaceholderVideo_(videoEl) {
+    if (!this.isManagedByPool_()) {
+      return false;
+    }
+    return !videoEl.classList.contains('i-amphtml-pool-media');
+  }
```

Comment:
@gmajoulet We may not need this since we only check that on buildCallback, where the mediapool probably didn't swap the video yet. If that's the case, we can remove this function.

Reply:
The media pool waits for the video build/layoutCallbacks to resolve, you can safely remove this check.

Reply:
SG, then this can be replaced with `this.isManagedByPool()`